lib/ddtrace/span.rb

```diff
@@ -288,8 +308,34 @@ def pretty_print(q)
       end
     end
 
+    # Return whether the duration is started or not
+    def started?
+      !@start_time.nil?
+    end
+
+    # Return whether the duration is finished or not.
+    def finished?
+      !@end_time.nil?
+    end
+
+    def complete?
```

Comment:
we shouldn't ever have an `end_time` without a `start_time`.

Since we call `start(finish_time || now) unless started?`, so `finished?` should be the same thing as `complete?` here, right?

Reply:
yup, updated